### ISSUE TITLE
Two equations* do not render in Pluto

### ISSUE THREAD

**Issue by mmikhasenko** [rjrosati/SymbolicTensors.jl#8]
![image](https://user-images.githubusercontent.com/22725744/148528178-ad3a9b35-e449-4244-bd0d-88743b26e649.png)

What is about `align`?

**Comment by rjrosati** [comment]
I haven't tested very much in Pluto. This is probably an issue with my override of `Base.show`

**Comment by mmikhasenko** [comment]
Yes, most likely. I thought it might work if stacking equations with align (in Base.show). Might be a bit tricky with closing the environment
